docs/sources/features/datasources/prometheus.md

```diff
@@ -50,7 +50,7 @@ Name | Description
 *Min step* | Set a lower limit for the Prometheus step option. Step controls how big the jumps are when the Prometheus query engine performs range queries. Sadly there is no official prometheus documentation to link to for this very important option.
 *Resolution* | Controls the step option. Small steps create high-resolution graphs but can be slow over larger time ranges, lowering the resolution can speed things up. `1/2` will try to set step option to generate 1 data point for every other pixel. A value of `1/10` will try to set step option so there is a data point every 10 pixels.
 *Metric lookup* | Search for metric names in this input field.
-*Format as* | **(New in v4.3)** Switch between Table & Time series. Table format will only work in the Table panel.
+*Format as* | **(New in v4.3)** Switch between Table, Time series or Heatmap. Table format will only work in the Table panel. Heatmap format is suitable for displaying metrics having histogram type on Heatmap panel. Under the hood, it converts cumulative histogram to regular and sorts series by the bucket bound.
```

Comment:
I think we can remove the notice about 4.3

Reply:
Yes, I also think it's outdated.